### PR TITLE
Monitor controller tests migration to RSpec

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

File: src/api/spec/controllers/webui/monitor_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      create_list(:status_history, 30, source: 'building', range: 10..42)
+      create_list(:status_history, 10, source: 'waiting', range: 10000..42000)
+      # i586
+      create_list(:status_history, 5, source: 'squeue_high', architecture: 'i586')
+      xhr :get, :events, {arch: 'x86_64', range: 8760}
+      @json_response = JSON.parse(response.body)
+    end
+
+    it { expect(@json_response['events_max']).to be <= 800 }
+    it { expect(@json_response['jobs_max']).to be <= 84000 }
```
Comment (bgeuken) [inline src/api/spec/controllers/webui/monitor_controller_spec.rb]: We can't check a fixed number here?

**Comment by bgeuken** [comment]
LGTM

File: src/api/spec/controllers/webui/monitor_controller_spec.rb (hunk 1 above)
Comment (mdeniz) [inline src/api/spec/controllers/webui/monitor_controller_spec.rb]: Nope... the number is as maximum the double of the upper part of the range... but is randomly generated, so you never are sure about having exactly that number in the values of the list

**Comment by hennevogel** [comment]
If it looks good then :ship: this :hankey:! :-)

**Pull request merged**